Test/Controller/ReservationsSpec.hs

```diff
@@ -64,18 +64,18 @@ tests = aroundAll (withIHPApp WebApplication config) do
                 count `shouldBe` 1
 
                 reservationJob <- query @ReservationJob |> orderByDesc #createdAt |> fetchOne
-                reservation <- fetch (get #reservationId reservationJob)
+                reservation <- fetch reservationJob.reservationId
                 -- Seat not assigned yet.
-                get #seatNumber reservation `shouldBe` 0
+                reservation.seatNumber `shouldBe` 0
 
 
                 -- Process job.
-                let frameworkConfig = getFrameworkConfig ?context
+                let frameworkConfig = ?context.frameworkConfig
```

Comment:
@mpscholten trying to fix the test with the new syntax to get the `frameworkConfig`, but when I try to [run it](https://github.com/Gizra/ihp-simple-seat-reservation#test) I get an error

![image](https://user-images.githubusercontent.com/125707/197605244-ed00a16e-01c4-4c73-b016-60aa5e5ce09c.png)

Reply:
Hm, could you do an `import IHP.Controller.RequestContext ()`?

Reply:
Doesn't work - same error

Reply:
Try `import IHP.Controller.RequestContext` without the `()` at the end. Locally this got it working 👍